tests/test_plugin.py

```diff
@@ -1,22 +1,23 @@
 """This modules includes unit tests for the plugin."""
 
-from six.moves import mock
-
-from delayed_assert import expect, assert_expectations
 import pytest
+from _pytest.config.argparsing import Parser
+from delayed_assert import expect, assert_expectations
+from reportportal_client.errors import ResponseError
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
Are these not all third party imports? I can split them however you'd like, but I don't think any of them are part of the standard library.

Reply:
reportportal_client is a local application.

Reply:
I still don't understand why that's the case since `reportportal_client` comes from `requirements.txt`, but I have reverted the line.

Reply:
Correct, that's my bad. I thought it is a common client where we are))